src/resources/views/crud/inc/show_tabbed_fields_on_the_fly.blade.php

```diff
@@ -0,0 +1,67 @@
+@php
```

Comment:
I override this field or we would end-up with alot of if else's because tabs ids must be different and we should tell show_fields that the field is beeing loaded on the fly.